packages/stream/media/media.js

```diff
@@ -0,0 +1,54 @@
+const media = (() => {
+  /** @type {MediaStreamConstraints} */
+  const defaultMediaConstraints = {
+    video: {
+      frameRate: 30,
+      width: { min: 640, ideal: 1280, max: 1280 },
+      height: { min: 360, ideal: 720, max: 720 },
+    },
+    audio: true,
+  }
+
+  /**
+   *
+   * @param {MediaStreamConstraints} mediaConstraints - The media stream constraints
+   */
+  const getUserMedia = async (mediaConstraints = {}) => {
+    const userMedia = await navigator.mediaDevices.getUserMedia({
+      video: defaultMediaConstraints.video,
+      audio: defaultMediaConstraints.audio,
+      ...mediaConstraints,
+    })
+
+    return userMedia
+  }
```

Comment:
@faiq-naufal I kinda confused on the usage of `mediaConstraints` part.
If look at this function, the audio is set by the `defaultMediaContraints.audio` (from above `defaultMediaConstraints` variable) which is the value is `true`.
However, on the PR's description also mention as per below :
```js
const mediaConstraints = {
  video: true,
  audio: true
};

const mediaStream = await InliveStream.media.getUserMedia(
  mediaConstraints
);
```

on the `mediaConstraints` from the description, the users also can add `audio: true` by themselves then (?), and will be added on this `getUserMedia` function which will be compiled with code `...mediaConstraints` (double `audio: true` ?).

Could you advice, so is it required for the users to input :
```js
const mediaConstraints = {
  video: true,
  audio: true
};
``` 
by themselves ?

Reply:
> the users also can add `audio: true` by themselves then (?)

Yes, the user can add the configuration constraints for audio and video by themselves. However, it is not necessary to input the configuration constraints because we have provided the default value as you mentioned above. The developers can set their own constraints configuration as an object and pass it as an argument for `getUserMedia`. If the developers set their own configuration for specific device (video or audio), it will override the default value and use the configuration constraints that the developers provided

Reply:
Okay, thanks for the confirmation mas.